From a84ddf84caefa2252428060901f4188334728113 Mon Sep 17 00:00:00 2001
From: Nicolin Chen <b42378@freescale.com>
Date: Fri, 30 Aug 2013 15:21:19 +0800
Subject: [PATCH 0177/1072] ENGR00277458-2 ARM: imx6q: Set pll3_pfd3_454m
 clock as spdif's parent

git://git.freescale.com/imx/linux-2.6-imx.git imx_3.10.17_1.0.0_beta
commit 5d00e279138fcf7d315c2d3d333a5678bb7a5750

Provisionally use pll3_pfd3_454m clock as spdif's parent clock, which can
fairly meet our playback and capture requirement.

Acked-by: Wang Shengjiu <b02247@freescale.com>
Signed-off-by: Nicolin Chen <b42378@freescale.com>
Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 arch/arm/mach-imx/clk-imx6q.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-imx/clk-imx6q.c b/arch/arm/mach-imx/clk-imx6q.c
index 738cd7c..9d37202 100644
--- a/arch/arm/mach-imx/clk-imx6q.c
+++ b/arch/arm/mach-imx/clk-imx6q.c
@@ -511,6 +511,7 @@ static void __init imx6q_clocks_init(struct device_node *ccm_node)
 	clk_set_parent(clk[ssi1_sel], clk[pll4_audio_div]);
 	clk_set_parent(clk[ssi2_sel], clk[pll4_audio_div]);
 	clk_set_parent(clk[ssi3_sel], clk[pll4_audio_div]);
+	clk_set_parent(clk[spdif_sel], clk[pll3_pfd3_454m]);
 
 	/* Set pll4_audio to a value that can derive 5K-88.2KHz and 8K-96KHz */
 	clk_set_rate(clk[pll4_audio_div], 541900800);
-- 
1.7.5.4

